src/main/java/seedu/address/model/ResumeBook.java

```diff
@@ -225,7 +225,7 @@ public void deleteInternship(Internship key) {
         internships.remove(key);
         for (Item item : resumes) {
             Resume resume = (Resume) item;
-            resume.getInternships().remove(id);
+            resume.getInternships().remove(Integer.valueOf(id));
```

Comment:
Does this do anything special or do you just want to be consistent with the signature of `remove`?

Reply:
It's to remove the value id, not the value at index id 💃